test/run.ts

```
@@ -0,0 +1,201 @@
+    .onExit((code) => {
+      console.log(`\nProcess exited with code: ${code}`);
+      process.exit(code);
+    })
+    .start();
+
+  // Automated steps
+  await sleep(2000);
+  runner.moveDown();
+  await sleep(500);
+  runner.enter();
+}
```

ESLint

[error] Expected "main" to come before "sleep".

(perfectionist/sort-modules)